tests/foreman/ui/test_host.py

```diff
@@ -2311,28 +2311,57 @@ def test_positive_gce_cloudinit_provision_end_to_end(
             googleclient.disconnect()
 
 
-@pytest.mark.destructive
 @pytest.mark.upgrade
-@pytest.mark.usefixtures('install_cockpit_plugin')
 @pytest.mark.tier2
-def test_positive_cockpit(session, default_sat):
+def test_positive_cockpit(destructive_sat):
```

Comment:
Okay, will use default_sat

Reply:
`destructive_sat` fixture name is a little misleading I think, here `destructive_sat` will spin up a separate isolated satellite to run this test on, which does not make a test destructive.
And I think, running `satellite-installer` on default_sat to enable cockpit here can keep the system in an unstable state for a while, and the same satellite can be used by xdist in automation to run other tests which might fail, so to keep automation safe we should run such tests on separate satellite, @JacobCallahan @jyejare can confirm or add some thoughts here?

Reply:
I kind agree on the definitions @Gauravtalreja1  mentioning if we are running satellite-installer on test does not mean the test is destructive or we are doing any destruction on the satellite as it's part of documentation run installer. I would rather get separate satellite as 'dedicated_satellite' or 'specialist_satellite'

Reply:
yes depends on the usage of xdist in our outomation
and I would like to see renamed marker as well

Reply:
Some nice thoughts here !

I agree that we should rename destructive_sat fixture to something `new_sat` and then all other fixtures can use access the `new_sat` fixtures, so be its `destructive_sat` or `katello_agent_sat` or be its any fixture! @JacobCallahan  I think its time to have a `new_sat` fixture that can be used by other fixtures!

Also, why we consider some actions destructive because we see that actions have the potential to destruct the satellite and that destruction can cause other tests running or the entire pipeline to fail. So actions like satellite-installer, foreman rake, foreman maintain have the potential to break the satellite and we have to look at these actions with that suspicious angle for being at safer site than destructive.

Coming back to the test here, then I surely consider this test as destructive because it's playing with adding a new service(`cockpit`) service also restarting the services on satellite. So I agree with the destructive_sat / new_sat approach here.